mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -236,7 +236,8 @@ class GBTClassifierSuite extends MLTest with DefaultReadWriteTest {
     sc.setCheckpointDir(path)
 
     val categoricalFeatures = Map.empty[Int, Int]
-    val df: DataFrame = TreeTests.setMetadata(data, categoricalFeatures, numClasses = 2)
+    val df: DataFrame =
+      TreeTests.setMetadata(data.map(_.toInstance), categoricalFeatures, numClasses = 2)
```

Comment:
This method should probably just accept LabeledPoints and map them internally, if almost all calls need the caller to map it